### ISSUE TITLE
Binga: Ball does not fall into basket

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#9681]
### Describe the bug

Download SWF file: [binga_decrypted.zip](https://github.com/ruffle-rs/ruffle/files/10810158/binga_decrypted.zip)

In Binga, the ball does not fall in Round 1, making it impossible to complete.
![image](https://user-images.githubusercontent.com/71368227/220818810-c110232a-a336-4eca-9f55-bf45b9bc0842.png)

Also, the timer does not count up and the Reset button does not work.

### Expected behavior

The ball should fall into the basket and the timer should count up.
![image](https://user-images.githubusercontent.com/71368227/220818911-dda52d4c-24c7-4c91-b230-02c5bf4cda24.png)

And the Reset button should reset the level.

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

Here are some messages seen in the terminal that may or may not be relevant:
```
WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.display.Graphics.beginGradientFill()
WARN render: ruffle_core::stub: Encountered stub: AVM2 flash.display.BitmapData.lock()
WARN render: ruffle_core::stub: Encountered stub: AVM2 flash.utils.Dictionary constructor with weak keys
ERROR render: ruffle_core::avm2::activation: TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: ::b2World_new)
ERROR render: ruffle_core::avm2::activation: TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: ::IsLocked)
```

**Comment by n0samu** [comment]
The error now is:
```
ERROR ruffle_core::avm2::events: Error dispatching event EventObject(EventObject { type: "render", class: flash.events::Event, ptr: 0x1cbc1f81c18 }) to handler FunctionObject(FunctionObject { ptr: 0x1cbc1e679c8 }) : Error: Error #2030: End of file was encountered.
        at cmodule.Box2D::FSM__start/work()
        at cmodule.Box2D::CRunner/work()
        at cmodule.Box2D::CLibInit/init()
        at Box2DAS.Common::b2Base$/initialize()
        at wck::WCK/create()
        at misc::Entity/ensureCreated()
        at MethodInfo-4556()
```

**Comment by n0samu** [comment]
Fixed by #12308
